in_toto/runlib.py

```diff
@@ -130,6 +130,11 @@ def record_artifacts_as_dict(artifacts):
             A list of file or directory paths used as materials or products for
             the link command.
 
+    follow_links: (optional)
+            Follow symbolic links if the linked dir exists (default is False).
+            NOTE: This parameter toggles following linked directories only,
```

Comment:
> Follow symbolic links if the **linked** dir exists (default is False).

I recommend you use "symlink" when referring to symbolic links, otherwise it clashes with in-toto's use of "link cmd," "link files," "Link Object," etc.

Reply:
Would it help to rename `follow_links` to something like `follow_symlink_dir`?  It might clear up the possible confusion with the default/different behavior with symlinked files.

Reply:
You should also consider adding a note about an infinite recursion possibility if a symlink points to a parent directory of itself.

Reply:
All good points, I will change as you suggest